easybuild/easyblocks/f/fluent.py

```diff
@@ -55,7 +55,12 @@ def build_step(self):
     def install_step(self):
         """Custom install procedure for FLUENT."""
 
-        cmd = "./INSTALL -noroot -silent -install_dir %s" % self.installdir
+        extra_args =''
+        # only include -noroot flag for older versions
+        if LooseVersion(self.version) < LooseVersion('15.0'):
+            extra_args += '-noroot'
+
+        cmd = "./INSTALL %s -debug -silent -install_dir %s %s" % (extra_args, self.installdir, self.cfg['installopts'])
```

Comment:
doesn't hurt, I guess

the real solution for large (debug) logs is supporting automatically zipping them at the end of the installation ;)